public/locations-list-map.js

```diff
@@ -406,60 +483,85 @@ function centerMapToState(map, state) {
   }
 }
 
-function centerMapToNearestMarkers(map, markers) {
```

Comment:
What’s happening here was that this was simply renamed to `centerMapToMarkersNearUser()` and removed the redundant (shared module level) variables map + markers, as they’re passed around but never change after init. Then I took a more “fundamental” (core/reusable) function created from the guts of THIS function and put that higher up, if that makes sense.

Reply:
overall I'm not a fan of module globals. But since you've already factored it that way, not sure it's worth the effort to go back to parameter passing.